public/main.js

```diff
@@ -12,6 +16,11 @@ if (isDev) {
 let mainWindow;
 
 function createWindow() {
+  expressApp.use(bodyParser.urlencoded({ extended: true }));
+  expressApp.use(bodyParser.json());
+
+  server.listen(3001);
```

Comment:
I’m curious to hear what @Donaldblanc was saying when he mentioned randomizing across open ports. My (perhaps unfounded) concern with choosing something like `3001` would be conflicts with other running processes.

Reply:
@conorsexton  on the next iteration when adding the ability to toggle the server on or off, we can have Node randomly choose and open port by placing a 0 as an argument in the server constructor.